federatedscope/db/accessor/data.py

```diff
@@ -100,7 +110,9 @@ def to_pb(self):
         return tablepb
 
     def from_pb(tablepb):
-        return Table(tablepb.name, Schema(tablepb.data.schema), protocol_to_pandas(tablepb.data.schema, tablepb.data.rows))
+        return Table(
+            tablepb.name, Schema(tablepb.data.schema),
+            protocol_to_pandas(tablepb.data.schema, tablepb.data.rows))
```

Comment:
Ideally, the class `Table` should be DBMS independent, since we will reuse it for mysql. 
So maybe we should move the function `from_pb` to accessor